### PR TITLE
*: move to klog v2

### PR DESCRIPTION
**Pull request by paulfantom**
Move to klog v2 for compatibility reasons.

cc @s-urbaniak

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/brancz/kube-rbac-proxy
 go 1.13
 
 require (
+	github.com/campoy/embedmd v1.0.0 // indirect
```
Comment (simonpasquier) [inline go.mod:6]: not sure why this is added?

Comment (paulfantom) [reply go.mod]: I run `make` and `go mod tidy` and it was added automatically. Removed it as it is not needed.

**Comment by brancz** [comment]
How come no tests were run against this? :thinking:

**Comment by paulfantom** [comment]
They were, but travis didn't report them back: https://travis-ci.org/github/brancz/kube-rbac-proxy/builds/733627457.

Maybe we should move to github actions?

**Comment by brancz** [comment]
Happy to move to github actions, but in a separate PR I would say. I retriggered the build.

**Comment by brancz** [comment]
lgtm on green

**Comment by s-urbaniak** [comment]
agreed with all above, lgtm too :+1:

**Pull request merged**